src/main/markdown/eco/eco-news.md

```diff
@@ -0,0 +1,165 @@
+GWT News
```

Comment:
I might try to investigate if we can automate this, by allowing libraries to publish some RSS feeds to some service where we can fetch and display them, libraries and apps might be able to include some RSS publishing right through the release process like github actions or other means.

Reply:
Would be awesome if we can automate this. To start, we can go with a manual solution.

Reply:
GitHub releases are probably the way to go. They can be consumed as Atom feed out of the box: https://github.com/gwtproject/gwt/releases.atom